src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star__blok">
+      <div class="star__line stars--0">
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
```

Comment:
There is a typo in the class name 'srar__style'. It should be 'star__style' to match the likely intended naming convention and ensure proper styling if the CSS is set up for 'star__style'.